### ISSUE TITLE
Questions about the information leak

### ISSUE THREAD

**Issue by Dutch-voyage** [daochenzha/ltsm#28]
Hi, dear authors, I appreciate your thorough work on exploring design choices of time series foundation models. However, noticing the significant improvement in comparison with baseline models, I doubt that the setting of TS prompt might influence the validity of the experiment results. 
Quite often, time series forecasting models use the statistics of train set (means and standard deviations) to normalize inputs. Using statistics of the validation set and test set is considered to bring about information leaks. A recent work on NIPS 2024 has made a similar mistake, mentioned in https://github.com/ForestsKing/GLAFF/issues/5 and other posts in their github repo issues. 
In this work, TS prompts (manually extracted features) are computed **on the whole validation set and test set**, which provide models with information of whole val/test set in evaluation.
(see promt_generation() in [ltsm](https://github.com/daochenzha/ltsm/tree/main)/[prompt_bank](https://github.com/daochenzha/ltsm/tree/main/prompt_bank)/[stat-prompt](https://github.com/daochenzha/ltsm/tree/main/prompt_bank/stat-prompt)
/prompt_generate_split.py)
![image](https://github.com/user-attachments/assets/7d801655-2199-4836-8525-82e349a64a2b)
It would be very appreciated if the authors could provide experiment results to address this concern. E.g. using TS prompts over a past period instead of the whole val/test set.

**Comment by LSC2204** [comment]
Thank you for pointing this out. We are aware of this concern and are actively working on addressing it in our codebase. Additionally, we have tested the model using prompts extracted from the training set instead, and the results are comparable. We will update the results accordingly and share them soon.

**Comment by Dutch-voyage** [comment]
Thank you for your reply, glad to hear that, looking forward to your updates!
